terraform/cloudfoundry/acl_router.tf

```diff
@@ -0,0 +1,32 @@
+resource "aws_network_acl_rule" "20_router" {
+    network_acl_id = "${aws_network_acl.cf.id}"
+    protocol = "tcp"
+    rule_number = 20
+    rule_action = "allow"
+    from_port = 80
+    to_port = 80
+    cidr_block = "10.0.0.0/20"
+    egress = false
+}
+
+resource "aws_network_acl_rule" "21_router" {
+    network_acl_id = "${aws_network_acl.cf.id}"
+    protocol = "tcp"
+    rule_number = 21
+    rule_action = "allow"
+    from_port = 443
+    to_port = 443
+    cidr_block = "10.0.0.0/20"
+    egress = false
+}
+
+resource "aws_network_acl_rule" "22_gorouter_replies" {
+    network_acl_id = "${aws_network_acl.cf.id}"
+    protocol = "tcp"
+    rule_number = 22
+    rule_action = "allow"
+    from_port = 1024
```

Comment:
@bleach and I had a chat about this. The ACLs are only really valuable if we have a plan to move the routers to a different (either the cell subnet or a completely new subnet) very shortly after, but I'm not sure what other implications that would have or when we'd do it. He's going to think about it and we'll talk about it together soon.

Could you hold off making changes for the other comments in the meantime?